inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/mq/CreatePulsarSubscriptionTaskListener.java

```diff
@@ -87,28 +88,29 @@ public ListenerResult listen(WorkflowContext context) throws WorkflowListenerExc
             topicBean.setTenant(tenant);
             topicBean.setNamespace(namespace);
             topicBean.setTopicName(topic);
-            List<String> pulsarClusters = PulsarUtils.getPulsarClusters(globalPulsarAdmin);
 
-            // Create a subscription in the Pulsar cluster (cross-region), you need to ensure that the Topic exists
-            for (String cluster : pulsarClusters) {
-                String serviceUrl = PulsarUtils.getServiceUrl(globalPulsarAdmin, cluster);
-                PulsarClusterInfo pulsarClusterInfo = PulsarClusterInfo.builder()
-                        .token(globalCluster.getToken()).adminUrl(serviceUrl).build();
-                try (PulsarAdmin pulsarAdmin = PulsarUtils.getPulsarAdmin(pulsarClusterInfo)) {
-                    boolean exist = pulsarOptService.topicIsExists(pulsarAdmin, tenant, namespace, topic);
-                    if (!exist) {
-                        String fullTopic = tenant + "/" + namespace + "/" + topic;
-                        log.error("topic={} not exists in {}", fullTopic, pulsarAdmin.getServiceUrl());
-                        throw new BusinessException("topic=" + fullTopic + " not exists in " + serviceUrl);
-                    }
+            // Create a subscription in the Pulsar cluster, you need to ensure that the Topic exists
+            try {
+                boolean exist = pulsarOptService.topicIsExists(pulsarAdmin, tenant, namespace, topic);
+                if (!exist) {
+                    String fullTopic = tenant + "/" + namespace + "/" + topic;
+                    String serviceUrl = pulsarClusterInfo.getAdminUrl();
+                    log.error("topic={} not exists in {}", fullTopic, serviceUrl);
+                    throw new BusinessException("topic=" + fullTopic + " not exists in " + serviceUrl);
+                }
 
-                    // Consumer naming rules: sortAppName_topicName_consumer_group
-                    String subscription = clusterBean.getAppName() + "_" + topic + "_consumer_group";
-                    pulsarOptService.createSubscription(pulsarAdmin, topicBean, subscription);
+                // Consumer naming rules: sortAppName_topicName_consumer_group
+                String subscription = clusterBean.getAppName() + "_" + topic + "_consumer_group";
+                pulsarOptService.createSubscription(pulsarAdmin, topicBean, subscription);
 
-                    // Insert the consumption data into the consumption table
-                    consumptionService.saveSortConsumption(groupInfo, topic, subscription);
-                }
+                // Insert the consumption data into the consumption table
+                consumptionService.saveSortConsumption(groupInfo, topic, subscription);
+            } catch (PulsarAdminException e) {
```

Comment:
The `PulsarAdminException` and `PulsarAdminException` can write in one line as follows:

```
catch (PulsarAdminException | BusinessException e)
```